doc/nims.rst

```diff
@@ -165,3 +196,136 @@ ends with ``.nims``:
   echo "hello world"
 
 Use ``#!/usr/bin/env -S nim --hints:off`` to disable hints.
+
+
+Benefits
+========
+
+Cross-Platform
+--------------
+
+It is a cross-platform scripting language that can run where Nim can run,
+e.g. you can not run Batch or PowerShell on Linux or Mac,
+the Bash for Linux might not run on Mac,
+theres no Unittests tools for Batch, etc.
```

Comment:
I think this is spelled "unit tests".

Reply:
also "theres" --> "there's", but since it is plural --> "there are no unit test tools"